imaginairy/schema.py

```diff
@@ -199,7 +222,8 @@ def __init__(self, prompt=None, **kwargs):
         # allows `prompt` to be positional
         super().__init__(prompt=prompt, **kwargs)
 
-    @validator("prompt", "negative_prompt", pre=True, always=True)
```

Comment:
I didn't really understand their explanation of the removal of "always" and what should be done to replicate behavior.
![Screenshot 2023-08-08 at 8 43 44 PM](https://github.com/brycedrennan/imaginAIry/assets/31525633/f8db248e-e242-46e4-a58a-cf3e7eb83548)

Reply:
Seems contradictory?:

![Screenshot 2023-08-09 at 3 14 03 AM](https://github.com/brycedrennan/imaginAIry/assets/31525633/b38edcb2-1c25-4380-8fd3-7e08751bd0e8)